private_set_intersection/javascript/src/implementation/client.ts

```diff
@@ -5,14 +5,18 @@ import { ERROR_INSTANCE_DELETED } from './constants'
 export type Client = {
   readonly delete: () => void
   readonly createRequest: (clientInputs: readonly string[]) => string
-  readonly processResponse: (
+  readonly getIntersection: (
+    serverSetup: string,
+    serverResponse: string
+  ) => number[]
```

Comment:
I'm not sure how this maps to the declarations in [private_set_intersection/javascript/custom_types/psi-cardinality/index.d.ts](#diff-c814943a4b0eb72c9f476ffabf1efb01)